### PR TITLE
modify description for flag usage 

### LINKED ISSUE #3038: the flag description for karmada-scheduler-estimator is wrong
**What happened**:

the flag description `--kubeconfig` for `karmada-scheduler-estimator` is wrong .

**What you expected to happen**:

Path to member cluster kubeconfig file

**How to reproduce it (as minimally and precisely as possible)**:

```console
karmada-scheduler-estimator --help
```
**Anything else we need to know?**:

The karmada-scheduler-estimator component use member cluster's kubeconfig, not the karmada control plane's kubeconfig

**Environment**:
- Karmada version: v1..4.1

### PR DESCRIPTION
**Pull request by Wang-Kai**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

modify description for flag usage

**Which issue(s) this PR fixes**:
Fixes #3038 

**Special notes for your reviewer**:

if new description is  not suitable, i will modify it again.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by Garrybest** [comment]
Nice finding, thanks a lot👍.

/lgtm

**Comment by Garrybest** [comment]
Hi @Wang-Kai, you need to sign the [DCO](https://github.com/karmada-io/karmada/pull/3039/checks?check_run_id=10548711088) first.

**Comment by Wang-Kai** [comment]
>

**Comment by Wang-Kai** [comment]
> Hi @Wang-Kai, you need to sign the [DCO](https://github.com/karmada-io/karmada/pull/3039/checks?check_run_id=10548711088) first.

I signed commit just now . But i don't know why the PR was closed, maybe the wrong button was clicked.

**Comment by RainbowMango** [comment]
Hi @Wang-Kai Just a reminder, you might need to add your email to [GitHub Emails](https://github.com/settings/emails), otherwise, GitHub can't recognize you and count your contributions.

File: cmd/scheduler-estimator/app/options/options.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	if o == nil {
 		return
 	}
-	fs.StringVar(&o.KubeConfig, "kubeconfig", o.KubeConfig, "Path to karmada control plane kubeconfig file.")
+	fs.StringVar(&o.KubeConfig, "kubeconfig", o.KubeConfig, "Path to member cluster's kubeconfig file.")
```
Comment (jwcesign) [inline cmd/scheduler-estimator/app/options/options.go:45]: I see there are two kinds of flag: `kubeconfig` and `karmada-kubeconfig`. `kubeconfig` make users confused(not sure its member clusters' kubeconfig to control plane's kubeconfig), so how about change the name, like `member-kubeconfig`? cc @lonelyCZ @XiShanYongYe-Chang 

And also, I see some components use `kubeconfig` as the control plane's kubeconfig: https://github.com/karmada-io/karmada/blob/07bb65bfe7990c3ff727640907f81e25615b4523/cmd/descheduler/app/options/options.go#L78, not unified.

Comment (RainbowMango) [reply cmd/scheduler-estimator/app/options/options.go:45]: > kubeconfig make users confused(not sure its member clusters' kubeconfig or control plane's kubeconfig),

Agree. But it is out of this PR's scope.

In addition, if we are going to rename the flag, we should deprecate the old one.

**Comment by RainbowMango** [comment]
/approve
All set now.
Thanks and welcome @Wang-Kai ~

**Pull request merged**